content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,232 @@
+:guilabel:`Sale Order` field and select the :abbr:`SO (Sales Order)` from which the product that is
+being repaired originated from. If a *Return* has been initiated for the product, select it from the
+drop-down in the :guilabel:`Return` field.
 
-Repairs from tickets
---------------------
+Choose an :guilabel:`Invoice Method` from the drop-down. Select :guilabel:`Before Repair` or
+:guilabel:`After Repair` to generate an invoice before or after the work is completed. Selecting
+:guilabel:`No Invoice` means that an invoice cannot be generated for this service.
 
-Clicking on *Repair* option, on your ticket page, a new repair order form is shown. Fill in the
-information as needed and choose the next step.
+If parts are required for the repair, they can be added in the :guilabel:`Parts` tab. Services can
+be added as product lines on the :guilabel:`Operations` tab. Additional information for the internal
+repair team can be added to the :guilabel:`Repair Notes` tab. Information for the customer can be be
+added to the :guilabel:`Quotation Notes` tab, and will be automatically added to the PDF of the
+quotations generated from this :guilabel:`Repair Reference`.
 
-.. image:: after_sales/aftersales5.png
+A :guilabel:`Repairs` smart button will be added to the ticket, linking to the repair order.
+
+.. image:: after_sales/after-sales-repair-smart-button.png
    :align: center
+   :alt: View of smart buttons focusing on repair button.
+
+.. note::
+   Once a user creates a repair order from a *Helpdesk* ticket, they will be able to access it
+   through the ticket's :guilabel:`Repair` smart button, or from a link in the :guilabel:`Chatter`,
+   even if they do not have access rights to the *Repair* application.
+
+Create a field service task from a ticket
+=========================================
 
-Plan onsite interventions from tickets
---------------------------------------
+On-site interventions can be planned from a ticket and managed through the *Field Service*
+application. Customers with portal access will be able to track the progress of a :guilabel:`Field
+Service` task just as they would a *Helpdesk* ticket.
+
+To create a new task, navigate to a :menuselection:`Helpdesk` ticket. Click :guilabel:`Create Task`
+to open the :guilabel:`Create a Field Service task` pop-up. Confirm or update the task
+:guilabel:`Title`.
+
+.. note::
+   The :guilabel:`Project` field on the :guilabel:`Create a Field Service task` pop-up will default
+   to the same Field Service project that was identified on the team's settings page. To change the
+   project for this specific task, select one from the :guilabel:`Project` field.
+
+   To change the default Field Service project for the team, go to :menuselection:`Helpdesk -->
+   Configuration --> Teams` to select a :guilabel:`Team`. Scroll to the :guilabel:`After-Sales`
+   section and choose new project under :guilabel:`Field Service`.
```

Comment:
should Field Service be italicized since it's an app name?

Reply:
yes, there is sometimes confusion because the project itself is also by default called "Field Service" so italicizing it will (hopefully) make it clearer 👍